pkg/collector/collector.go

```diff
@@ -212,8 +212,6 @@ func (r *ChaosCollector) setUnarchivedExperiment(req ctrl.Request, obj v1alpha1.
 		archive.Action = string(chaos.Spec.Action)
 	case *v1alpha1.IoChaos:
 		archive.Action = string(chaos.Spec.Action)
-	case *v1alpha1.DNSChaos:
-		archive.Action = string(chaos.Spec.Action)
```

Comment:
Why do we delete this branch for `*v1alpha1.DNSChaos`? I think we should keep it.

Reply:
Nevermind.... This is an outdated file.  😰

Reply:
it's added in another pr https://github.com/chaos-mesh/chaos-mesh/pull/1238